autotest/gcore/tiff_read_subifds.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env pytest
+# -*- coding: utf-8 -*-
+###############################################################################
+# $Id$
+#
+# Project:  GDAL/OGR Test Suite
+# Purpose:  Test basic read support for tiff files using overviews in subifds
+# Author:   Thomas Bonfort <thomas.bonfort@airbus.com>
+#
+###############################################################################
+# Copyright (c) 2019,  Thomas Bonfort <thomas.bonfort@airbus.com>
+#
+# This library is free software; you can redistribute it and/or
```

Comment:
Can you use the X/MIT text instead (as in mask.py) ? LGPL use in other files is a bit of historical accident.